detectron2/data/datasets/coco.py

```diff
@@ -262,12 +263,10 @@ def file2id(folder_path, file_path):
 
     dataset_dicts = []
     for (img_path, gt_path) in zip(input_files, gt_files):
+        w, h = imagesize.get(gt_path)
```

Comment:
with this we probably lose the flexibility of `PathManager` ?
For me for example I've registred multiple new handlers for PathManager and my images are all online.

maybe add `local_path = PathManager. get_local_path(gt_path)` before ?

Reply:
well that sounds good :)

Reply:
@bendidi Updated in a31f70b